docs/detections/rules-ui-manage.asciidoc

```diff
@@ -186,16 +188,17 @@ To export and import detection rules:
 .. Select *Bulk actions* -> *Export*, then save the exported file.
 . To import rules:
 +
-NOTE: You need at least `Read` privileges for the `Action and Connectors` feature to import rules with actions. If you're importing rules without actions, `Action and Connectors` feature privileges are not required. Refer to <<enable-detections-ui>> for more information.
+NOTE: To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To import rules with actions and overwrite an existing connector, you need `All` privileges for the `Action and Connectors` feature. If you're importing rules without actions, `Action and Connectors` feature privileges are not required. Refer to <<enable-detections-ui>> for more information.
```

Comment:
```suggestion
NOTE: To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To import rules with actions and overwrite an existing connector, you need `All` privileges for the `Actions and Connectors` feature. If you're importing rules without actions, `Actions and Connectors` feature privileges are not required. Refer to <<enable-detections-ui>> for more information.
```
My suggestions from above would also apply here if you decide to go with them.

Reply:
```suggestion
NOTE: To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To overwrite or add new connectors, you need `All` privileges for the `Actions and Connectors` feature. To import rules without actions,  you don't need `Actions and Connectors` privileges. Refer to <<enable-detections-ui>> for more information.
```